test/unit/com/palantir/cassandra/utils/SchemaAgreementCheckTest.java

```diff
@@ -165,7 +172,8 @@ public void checkSchemaAgreement_failsIfEndpointsAreMissing()
 
         SchemaAgreementCheck schemaAgreementCheck = new SchemaAgreementCheck(() -> schema,
                                                                              () -> ImmutableMap.of(InetAddresses.forString("127.0.0.1"), state).entrySet(),
-                                                                             InetAddresses.forString("127.0.0.1"));
+                                                                             InetAddresses.forString("127.0.0.1"),
+                                                                             true);
```

Comment:
Add a unit test for the requirePeer flag?

Reply:
[1618109](https://github.com/palantir/cassandra/pull/647/commits/161810938ecb3e09d372d9442042654e2ef96226)